src/routes/versions.rs

```diff
@@ -156,7 +156,7 @@ pub async fn version_get(
 #[delete("{version_id}")]
 pub async fn version_delete(
     req: HttpRequest,
-    info: web::Path<(models::ids::ModId, models::ids::VersionId)>,
+    info: web::Path<(models::ids::VersionId,)>,
     pool: web::Data<PgPool>,
 ) -> Result<HttpResponse, ApiError> {
     check_is_moderator_from_headers(
```

Comment:
This should allow users to delete their own mod versions if they have the permissions